packs/classfeatures/eternal-confidence.json

```
@@ -24,10 +24,24 @@
             "remaster": false,
             "title": "Pathfinder Advanced Player's Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "eternal-confidence",
```

```suggestion
                "option": "finisher:confident",
```
no need to make a new roll option when one already exists.

---

I didn't want to use that to avoid misfiring if future or 3pp stuff predicates something on confident finisher. Would you be happier with using this
```json
{
	"and": [
		"feature:eternal-confidence",
		{
			"or": [
				"finisher",
				"opportune-riposte"
			]
		}
	]
}
```
over `"eternal-confidence"` in Confident Finisher and Precise Finisher instead? This also wouldn't need a new roll option, while staying accurate to the feat.

---

It's fine as is then